### ISSUE TITLE
Expose parameters to User

### ISSUE THREAD

**Issue by j48zheng** [HoloClean/HoloClean-Legacy-deprecated#144]
Expose the following in softmax for user to set: 
learning rate, 
pruning threshold, 
number of iterations
